test/recipes/15-test_ec.t

```diff
@@ -16,7 +16,7 @@ use OpenSSL::Test::Utils;
 
 setup("test_ec");
 
-plan skip_all => 'EC is not supported in this build' if disabled('ec');
+plan skip_all => 'EC is not supported in this build' if (disabled('ec') || disabled('ecx'));
```

Comment:
This is wrong. You need to split the testcases and with the no-ecx skip the ecx ones only.